unstructured_api_tools/pipelines/templates/pipeline_api.txt

```diff
@@ -29,11 +40,33 @@ async def pipeline_1(request: Request, file: UploadFile = File(),
 {{param}}: List[str] = Form(default=[]),
 {% endfor %}
 ):
-
+    
+    media_type = request.headers.get("accept")
+    {% if response_type %}
+    default_response_type = "{{response_type}}"
+    media_type = default_response_type if media_type == "*/*" else media_type
+    {% endif %}
     text = file.file.read().decode("utf-8")
+
+    {% if response_type %}
+    response = pipeline_api(text, {% for param in multi_string_param_names %}m_{{param}}={{param}}, {% endfor %} response_type=media_type,)
+    {% else %}
     response = pipeline_api(text, {% for param in multi_string_param_names %}{{param}}, {% endfor %})
+    {% endif %}
 
-    return response
+    if _is_confict_response_type(media_type, type(response)):
+        return PlainTextResponse(
+            content=f"Conflict in media type {media_type} with response type {type(response)}.\n",
+            status_code=status.HTTP_409_CONFLICT,
```

Comment:
Perhaps a `406` instead? A 409 Conflict tends to be more about existing resources that the server is aware of.